engine/src/main/java/org/terasology/rendering/logic/MeshRenderer.java

```diff
@@ -191,23 +190,22 @@ private void renderEntitiesByMaterial(SetMultimap<Material, EntityRef> meshByMat
                     MeshComponent meshComp = entity.getComponent(MeshComponent.class);
                     LocationComponent location = entity.getComponent(LocationComponent.class);
 
-                    if (isHidden(entity, meshComp) || location == null || Float.isNaN(location.getWorldPosition().x)|| meshComp.mesh == null || !isRelevant(entity, location.getWorldPosition())) {
+                    if (isHidden(entity, meshComp) || location == null || Float.isNaN(location.getWorldPosition().x)|| meshComp.mesh == null || !isRelevant(entity, JomlUtil.from(location.getWorldPosition()))) {
                         continue;
                     }
                     if (meshComp.mesh.isDisposed()) {
                         logger.error("Attempted to render disposed mesh");
                         continue;
                     }
 
-                    location.getWorldRotation(worldRot);
-                    location.getWorldPosition(worldPos);
+                    worldRot.set(JomlUtil.from(location.getWorldRotation()));
```

Comment:
I really like this change... I'm always confused by "getter" methods that actually mutate the method argument o.O

Reply:
The "dest" parameter is common practice in joml though. It's purpose is not to have an unnecessary memory allocation.